cli/src/main/java/com/devonfw/tools/ide/tool/helm/HelmUrlUpdater.java

```diff
@@ -26,7 +26,30 @@ protected String getVersionPrefixToRemove() {
   @Override
   protected String getGithubOrganization() {
 
-    return "helm";
+    return getTool();
```

Comment:
The name of the tool and the github organization may not be the same thing.
Such change is not wrong but also not really an improvement and especially not related to the actual story.